src/I18n/I18n.php

```diff
@@ -246,6 +238,20 @@ public static function defaultLocale()
     }
 
     /**
+     * Set default fallback domain. If a domain can't provide a translation
+     * messages from the fallback domain are used.
+     *
+     * Get the currently set domain if argument is null.
+     *
+     * @param string|null $name name of the fallback-translator
+     * @return string
+     */
+    public static function defaultFallbackDomain($name = null)
```

Comment:
Sorry, wasn't clear. That's what I meant, too:

- domain fallback stays in and keeps using the "default" domain (current CakePHP behavior)
- no setter to change the default domain (current CakePHP, but must be removed from the PR)
- introduce only one new setter `useFallback($enable = true)` to toggle the fallback on/off